public/team.yaml

```diff
@@ -114,36 +114,31 @@ institutions:
         picture: img/team/ChrisMungall.jpg
         title: Monarch PI
         alumni: false
-        bio: "Dr. Mungall is currently a PI of the Monarch Initiative. He has a BSc in Computer Science from the University of Edinburgh and a Ph.D. from University of Edinburgh in Biological Sciences. He is currently the PI of the Monarch Initiative and his current research focuses on the use of informatics techniques and tools to integrate and interpret life science data across a variety of domains. In particular, he is interested in connecting biological models at multiple scales: the biochemical, molecular, cellular, organismal and ecological."
+        bio: "Dr. Mungall is a PI of the Monarch Initiative. He has a Ph.D. from University of Edinburgh in Bioinformatics. He leads a group working on the application of computational techniques to problems in life sciences of relevance to the health of humans and the health of the planet. His main interest is the application of artificial intelligence, knowledge-based methods, and biocuration to advance our understanding of the interconnected role of genes and genetic mechanisms in key biological processes.
       - name: Seth Carbon
         picture: img/team/SethCarbon.png
-        title: Software Engineer
+        title: Software engineer
         alumni: false
         bio: Seth is an occasional software engineer for the Monarch Initiative. He specializes in tools for science and education and has participated in projects in a broad range of fields, including Bioinformatics, Planetary Science, Geology, and Linguistics. His current focus is on capturing and making use of ontology and annotation data. His educational background is undergraduate degrees in Mathematics and Computer Science (UCSC).
-      - name: Eric Douglass
-        picture: img/team/EricDouglass1.jpg
-        title: Software Engineer
-        alumni: false
-        bio: Eric is a software engineer that works with Seth and Chris on data validation pipelines, Turtle and Ontologies in triplestores and other tools. He comes from a background originally in physics from Lewis and Clark college and then software development in industry, Eric got his first taste of ontologies at his first tech job in Portland.
-      - name: Nathan Dunn
-        picture: img/team/NathanDunn.jpg
-        title: Software Engineer
-        alumni: false
```

Comment:
I think rather than deleting Eric, you just want to set alumni: true

Reply:
i wasn't sure he ever actually worked on monarch, but ok

Reply:
Oh, I may be wrong then.